vic/drivers/shared_image/include/vic_driver_shared_image.h

```diff
@@ -201,7 +212,7 @@ void vic_nc_info(nc_file_struct *nc_hist_file, out_data_struct **out_data,
                  nc_var_struct *nc_vars);
 void vic_restore(void);
 void vic_start(void);
-void vic_store(void);
+void vic_store(dmy_struct *dmy_current);
```

Comment:
Hmm, I think we want to delete this input parameter (since it's unused in `vic_store`)?